ebean-ddl-generator/src/main/java/io/ebeaninternal/dbmigration/DefaultDbMigration.java

```diff
@@ -805,58 +789,18 @@ private File modelDirectory(File migrationDirectory) {
    */
   protected DatabasePlatform platform(Platform platform) {
     switch (platform) {
-      case H2:
```

Comment:
Plattform is an enum and not easily extendable, do you think it makes sense to have a Platform.USER1 that can be used for custom implementations?

Reply:
I think that is an excellent suggestion.  Perhaps Platform.CUSTOM or Platform.USERDEFINED ...

Reply:
+1 for CUSTOM